tools/figma-inspector/backend/utils/property-parsing.ts

```diff
@@ -876,6 +877,44 @@ export async function generateTextSnippet(
                         );
                     }
                     break;
+                case "horizontal-alignment":
+                    if (
+                        "textAlignHorizontal" in sceneNode &&
+                        typeof sceneNode.textAlignHorizontal === "string"
+                    ) {
+                        let slintValue: string | null = null;
+                        switch (sceneNode.textAlignHorizontal) {
+                            case "LEFT":
+                                slintValue = "left";
+                                break;
+                            case "CENTER":
+                                slintValue = "center";
+                                break;
+                            case "RIGHT":
+                                slintValue = "right";
+                                break;
+                            case "JUSTIFIED":
+                                slintValue = "justify"; // Slint's justify might differ
```

Comment:
There is no support for text justification in Slint. I think it would be better to not map or produce an error, instead of generating Slint code that won't compile.